roles/edpm_update/tasks/bootc.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Update bootc image
+  ansible.builtin.shell: bootc switch "{{ edpm_bootc_os_container_image }}"
```

Comment:
How are service container images updated? Are they logically bound and pre-fetched in this image?

Reply:
https://github.com/openstack-k8s-operators/edpm-ansible/pull/905#discussion_r2004802217